roles/xmpp/tasks/prosody.yml

```diff
@@ -37,6 +34,23 @@
   template: src=prosody.cfg.lua.j2 dest=/etc/prosody/prosody.cfg.lua group=root owner=root
   notify: restart prosody
 
+- name: Copy SSL private key and cert
```

Comment:
Permission problems.  The server runs as user/group prosody.  This could be solved with a group to access /etc/letsencrypt, but I wasn't comfortable whacking permissions on /etc/letsencrypt since I don't know what the client does after initial setup.

This is something to consider when we're evaluating other clients.